### ISSUE TITLE
Perplexity for data ordering pattern for Neural Machine Translation

### ISSUE THREAD

**Issue by Bachstelze** [facebookresearch/XLM#206]
In the recent paper  [Data Ordering Patterns for Neural Machine Translation:An Empirical Study](https://arxiv.org/pdf/1909.10642.pdf) the ordering gains +1.8 BLEU on the  IWSLT2015 English-Vietnamese language data set. The pretrained xlm could be perfect for preprocessing the training data for a large scale NMT. And there is already a function in the `evaluator.py`  which calculates the perplexity:

    def evaluate_clm(self, scores, data_set, lang1, lang2):
        """
        Evaluate perplexity and next word prediction accuracy.
        """

But i didn't found the preprocessing scripts, neither the parallel data for NMT in this repo...
